RELEASES.md

```diff
@@ -1,3 +1,45 @@
+## 1.16
+
+### For iOS and Android
+
+### Features
+
+- EIP-1159 support ([details](https://github.com/status-im/status-react/pull/12369))
```

Comment:
Wouldn't it be neater to use PR numbers in the links instead of "details"? Like this:
```suggestion
- EIP-1159 support ([#12369](https://github.com/status-im/status-react/pull/12369))
```

Reply:
Yeah what do you think @churik? Let's define this so I can merge it

Reply:
agree, that's seems nice update